packages/notifications-library/tests/conftest.py

```diff
@@ -84,6 +90,14 @@ def user_data(
     )
 
 
+@pytest.fixture
+def sharer_data(user_name: str, message: str = "") -> SharerData:
```

Comment:
having default fixtures is not a nice practice. Either create  a separate `message` fixture or fill it with a fake i.e.
```python
from faker import Faker

@pytest.fixture
def sharer_data(user_name: str, faker: Faker) -> SharerData:
    return SharerData(
        user_name=user_name,
        message=faker.sentence(),
    )

```

Reply:
merci